### PR TITLE
PLT-3005 Added confirmation screen to integrations

### PR DESCRIPTION
**Pull request by DavidLu1997**
#### Summary
When creating an integration, a confirmation screen now appears with the integration's URL/token, along with a link to appropriate documentation.

#### Ticket Link
https://mattermost.atlassian.net/browse/PLT-3005

#### Checklist
- [x] Has UI changes
- [x] Includes text changes and localization files updated

### CODE REVIEW

**Comment by coreyhulen** [comment]
+1, needs a rebase

**Comment by enahum** [comment]
this also needs to be applied to the new OAuth 2.0 provider

File: webapp/components/integrations/components/confirm_integration.jsx [hunk 1]
```diff
@@ -0,0 +1,108 @@
...
+        } else if (this.state.type === Constants.Integrations.OUTGOING_WEBHOOK) {
+            textId = 'add_outgoing_webhook';
+        }
+
+        return (
+            <div className='backstage-content row'>
+                <BackstageHeader>
+                    <Link to={'/' + this.props.team.name + '/integrations/' + this.state.type}>
+                        <FormattedMessage
+                            id={'installed_' + this.state.type + '.header'}
```
Comment (enahum) [inline webapp/components/integrations/components/confirm_integration.jsx]: I know this works, but actually you shouldn't use dynamic id's for Localizations. 

Can you please change this?

**Comment by DavidLu1997** [comment]
@enahum added OAuth 2.0 Apps, used static IDs

Also added back `getUrlParameters` to `Utils.jsx`

**Comment by enahum** [comment]
+1

File: webapp/components/integrations/components/add_oauth_app.jsx [hunk 2]
```diff
@@ -144,8 +144,8 @@ export default class AddOAuthApp extends React.Component {
 
         OAuthActions.registerOAuthApp(
             app,
-            () => {
-                browserHistory.push('/' + this.props.team.name + '/integrations/oauth2-apps');
+            (data) => {
+                browserHistory.push('/' + this.props.team.name + '/integrations/confirm?type=oauth2-apps&token=' + data.id + '&secret=' + data.client_secret + '&callback_urls=' + data.callback_urls.join(', '));
```
Comment (crspeller) [inline webapp/components/integrations/components/add_oauth_app.jsx]: Passing around secrets in the URL? Probably should figure out a better way to do this.

**Comment by DavidLu1997** [comment]
Closing to resubmit by rewriting part of API

**Pull request closed without merging**